components/dashboard/src/onboarding/OnboardingStep.tsx

```diff
@@ -54,7 +60,18 @@ export const OnboardingStep: FC<Props> = ({
 
                 {error && <Alert type="error">{error}</Alert>}
 
-                <div className="mt-4">
+                <div className={"mt-4" + (onCancel ? " flex space-x-2" : "")}>
```

Comment:
**suggestion**: Minor, but could we use a block button here (see **[relevant component property](https://github.com/gitpod-io/gitpod/blob/bbc0985c67f4ac089018edb03e0df70f803f3a6c/components/dashboard/src/components/Button.tsx#L15)**) and order the secondary button below?

| BEFORE | AFTER |
|-|-|
| <img width="1558" alt="Screenshot 2023-04-26 at 17 55 24" src="https://user-images.githubusercontent.com/120486/234616075-98bc3448-3e04-41d0-b9ff-ab5ef5a636ed.png"> | <img width="1558" alt="Screenshot 2023-04-26 at 17 55 21" src="https://user-images.githubusercontent.com/120486/234616118-4e2cbc7d-cff8-491e-bbd3-5ecafc9b050c.png"> |

Reply:
I'm not a designer, but is that better? 🙈 😇

Reply:
😂  

This elevates the primary action in a vertical order to avoid mixing the secondary action and can help provide balance in mobile layouts. In this case it seems a bit stretched because of the the width limit but should be fine to merge as is. You call. 🏓